build-system/tasks/e2e/describes-e2e.js

```diff
@@ -117,12 +140,30 @@ async function createSelenium(opt_config = {}) {
     'args': args,
   };
   capabilities.set('chromeOptions', chromeOptions);
-
   const builder = new Builder().withCapabilities(capabilities);
   const driver = await builder.build();
   return driver;
 }
 
+/**
+ * Configure a firefox driver.
+ *
+ * @param {!SeleniumConfigDef} config
+ * @return {!SeleniumDriver}
+ */
+async function createFirefoxDriver(config) {
```

Comment:
It is possible to consolidate this with `createChromeDriver()` or at least make them more similar?

Reply:
done. both now just create `args` array